internal/report/report.go

```diff
@@ -148,14 +192,29 @@ func newPolicyReportResult(policy policiesv1.Policy, resource unstructured.Unstr
 		Kind:    constants.KubewardenKindAdmissionPolicy,
 	}:
 		name = "ap-" + policy.GetName()
-	default:
-		// this never happens
+	default: // this never happens
 		log.Fatal().Msg("no policy found when creating PolicyReportResult")
 	}
 
 	time := metav1.Now()
 	timestamp := *time.ProtoTime()
 
+	var severity v1alpha2.PolicyResultSeverity
+	var scored bool
+	if policy.GetPolicyMode() == policiesv1.PolicyMode(policiesv1.PolicyModeStatusMonitor) {
+		scored = true
+		severity = SeverityInfo
+	}
+
+	var category string
```

Comment:
Category sadly is a string. It would be nice to have several overlapping categories: mutating, validating, context-aware, and so on.

policy-reporter UI allows filtering by them:

![categories](https://github.com/kubewarden/audit-scanner/assets/2196685/c4636265-1d24-4e70-96d3-4bb04ad63d5c)

Reply:
See the message from above, let's try to understand what is the purpose/logic behind `policies.kyverno.io/category` annotation. Then we can just read it from the policy annotation